web/ui/templates/targets.html

```diff
@@ -6,7 +6,7 @@ <h2 id="targets">Targets</h2>
       <table class="table table-condensed table-bordered table-striped table-hover">
         {{range $job, $pool := .Pools}}
           <thead>
-            <tr><th colspan="5" class="job_header">{{$job}}</th></tr>
```

Comment:
This is not valid HTML5. Please replace `name` with `href="#...` and define an `id` attribute instead.